whois-api/src/main/java/net/ripe/db/whois/api/rdap/VCardBuilder.java

```diff
@@ -1,97 +1,116 @@
 package net.ripe.db.whois.api.rdap;
 
 import com.google.common.base.Joiner;
-import com.google.common.collect.ImmutableMap;
 import com.google.common.collect.Lists;
 import net.ripe.db.whois.api.rdap.domain.vcard.VCard;
 import net.ripe.db.whois.api.rdap.domain.vcard.VCardKind;
-import static net.ripe.db.whois.api.rdap.domain.vcard.VCardName.EMAIL;
-import static net.ripe.db.whois.api.rdap.domain.vcard.VCardName.ADDRESS;
-import static net.ripe.db.whois.api.rdap.domain.vcard.VCardName.KIND;
-import static net.ripe.db.whois.api.rdap.domain.vcard.VCardName.FN;
-import static net.ripe.db.whois.api.rdap.domain.vcard.VCardName.GEO;
-import static net.ripe.db.whois.api.rdap.domain.vcard.VCardName.TELEPHONE;
-import static net.ripe.db.whois.api.rdap.domain.vcard.VCardName.VERSION;
-import static net.ripe.db.whois.api.rdap.domain.vcard.VCardName.ORG;
-import static net.ripe.db.whois.api.rdap.domain.vcard.VCardType.TEXT;
-import static net.ripe.db.whois.api.rdap.domain.vcard.VCardType.URI;
 import net.ripe.db.whois.api.rdap.domain.vcard.VCardName;
-import net.ripe.db.whois.api.rdap.domain.vcard.VCardType;
 import net.ripe.db.whois.api.rdap.domain.vcard.VCardProperty;
+import net.ripe.db.whois.api.rdap.domain.vcard.VCardType;
 import net.ripe.db.whois.common.domain.CIString;
-import static java.util.Collections.nCopies;
 
+import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
 import java.util.Set;
 
+import static java.util.Collections.nCopies;
+import static net.ripe.db.whois.api.rdap.domain.vcard.VCardName.ADDRESS;
+import static net.ripe.db.whois.api.rdap.domain.vcard.VCardName.EMAIL;
+import static net.ripe.db.whois.api.rdap.domain.vcard.VCardName.FN;
+import static net.ripe.db.whois.api.rdap.domain.vcard.VCardName.GEO;
+import static net.ripe.db.whois.api.rdap.domain.vcard.VCardName.KIND;
+import static net.ripe.db.whois.api.rdap.domain.vcard.VCardName.ORG;
+import static net.ripe.db.whois.api.rdap.domain.vcard.VCardName.TELEPHONE;
+import static net.ripe.db.whois.api.rdap.domain.vcard.VCardName.VERSION;
+import static net.ripe.db.whois.api.rdap.domain.vcard.VCardType.TEXT;
```

Comment:
We do not have to include all the vcard properties in json only those that RFC says  must or those that we know for sure.